### ISSUE TITLE
TypeError on GH Pages page load

### ISSUE THREAD

**Issue by ryanlovett** [executablebooks/mystmd#1178]
## Description
We have a MyST site deployed at https://jupyterhealth.github.io/myst-site-book/ via [github actions](https://github.com/jupyterhealth/myst-site-book/actions/workflows/deploy.yml). I can visit the front page of the site, but if I click on About or Projects in the top navigation, I see the error:

```
Application Error
TypeError: Cannot read properties of undefined (reading 'frontmatter')
    at E (https://jupyterhealth.github.io/myst-site-book/build/_shared/chunk-OQ6RQI2G.js:4:1714)
    at Po (https://jupyterhealth.github.io/myst-site-book/build/_shared/chunk-XJN3BT5Q.js:6:19476)
    at Vu (https://jupyterhealth.github.io/myst-site-book/build/_shared/chunk-XJN3BT5Q.js:8:3134)
    at mc (https://jupyterhealth.github.io/myst-site-book/build/_shared/chunk-XJN3BT5Q.js:8:44439)
    at fc (https://jupyterhealth.github.io/myst-site-book/build/_shared/chunk-XJN3BT5Q.js:8:39465)
    at Rd (https://jupyterhealth.github.io/myst-site-book/build/_shared/chunk-XJN3BT5Q.js:8:39396)
    at ul (https://jupyterhealth.github.io/myst-site-book/build/_shared/chunk-XJN3BT5Q.js:8:39256)
    at Zu (https://jupyterhealth.github.io/myst-site-book/build/_shared/chunk-XJN3BT5Q.js:8:35678)
    at sc (https://jupyterhealth.github.io/myst-site-book/build/_shared/chunk-XJN3BT5Q.js:8:34634)
    at Il (https://jupyterhealth.github.io/myst-site-book/build/_shared/chunk-XJN3BT5Q.js:1:8284)
```

If i reload that page, the error goes away. If I click on About while I am looking at /about, I will see the same error.

This is not reproducible running a local server, in part because the repo is not the org's main gh pages site. For example the site nav uses the url /myst-site-book/about which references the path to the repo and to the page within the repo. However this path does not exist when running on localhost. One must manually enter localhost:{port}/about to visit that page. When I do so, I don't see the error.

**Comment by ryanlovett** [comment]
I've tried pinning mystmd in GitHub actions to something older (1.1.56, 1.1.55), but the issue is still present.

**Comment by rowanc1** [comment]
Taking a look now.

**Comment by rowanc1** [comment]
We were using an internal pre-fetch link, but that only works on non-static pages! This changes to using the correct type of link. I will deploy a fix shortly and let you know when you can re-deploy!

**Comment by agoose77** [comment]
I think this has been fixed by #378! Please re-open if not.
